app/models/practice_editor.rb

```diff
@@ -0,0 +1,27 @@
+class PracticeEditor < ApplicationRecord
+  include VaEmail
+
+  belongs_to :practice
+  belongs_to :user
+
+  attr_accessor :email
+
+  # Add in a failsafe to prevent a practice from having no associated practice editors
+  before_destroy do
+    ensure_practice_has_at_least_one_practice_editor unless self.destroyed_by_association
+  end
+
+  validates :email, presence: true, format: valid_va_email, on: :create
+
+  def self.create_and_invite(practice, user, email)
```

Comment:
we don't need to pass in email here now right? since we removed the `email` field from the `PracticeEditor` table

Reply:
Right, but we do still need to include it on line `17`, because if we don't, the validation won't work because the email would not be present. Does that make sense. So it would be something link:

```
self.create!(practice: practice, user: user, email: user.email)self.create!(practice: practice, user: user, email: user.email)
```